### PR TITLE
[Bugfix] #17893 - Fixes the Cybersuit construction recipe for machinists. Lets them have access.

### PR DESCRIPTION
**Pull request by Eyeveri**
- Creates a version of the cybersuit so machinists can use it. 

Bugfix for [#17893](https://github.com/Aurorastation/Aurora.3/issues/17893)

### CODE REVIEW

**Comment by Eyeveri** [comment]
!bugfix

File: code/modules/clothing/spacesuits/rig/rig_construction.dm [hunk 1]
```diff
@@ -154,6 +154,15 @@
 	board_type = /obj/item/circuitboard/rig_assembly/illegal/hacker
 	target_board_type = /obj/item/circuitboard/rig_assembly/illegal/targeting/hacker
 
+/obj/item/rig_assembly/combat/illegal/hacker/machinist
+	name = "cybersuit control module assembly"
+	desc = "The assembly frame for an advanced powered armor suit with many cyberwarfare enhancements. Comes with built-in insulated gloves for safely tampering with electronics."
+	icon_base = "hacker"
+	icon_state = "hacker1"
```
Comment (FluffyGhoster) [inline code/modules/clothing/spacesuits/rig/rig_construction.dm:161]: Don't need to set them, if they're the same as the parent

File: code/modules/clothing/spacesuits/rig/rig_construction.dm [hunk 2]
```diff
@@ -154,6 +154,15 @@
...
+	rig_type = /obj/item/rig/light/hacker/machinist
+	board_type = /obj/item/circuitboard/rig_assembly/illegal/hacker
+	target_board_type = /obj/item/circuitboard/rig_assembly/illegal/targeting/hacker
```
Comment (FluffyGhoster) [inline code/modules/clothing/spacesuits/rig/rig_construction.dm:164]: Don't need to set them, if they're the same as the parent

**Comment by NonQueueingMatt** [comment]
!review+

File: code/modules/clothing/spacesuits/rig/rig_construction.dm (hunk 1 above)
Comment (Eyeveri) [reply code/modules/clothing/spacesuits/rig/rig_construction.dm:161]: Not sure what this is asking to do - do you want this removed from the file? I just did a copy and paste of the recipe.

**Comment by NonQueueingMatt** [comment]
Reopen when this is fixed.

**Pull request closed without merging**